ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -300,8 +310,10 @@ public RouterConfig(VerifiableProperties verifiableProperties) {
         verifiableProperties.getIntInRange("router.scaling.unit.max.connections.per.port.plain.text", 5, 1, 100);
     routerScalingUnitMaxConnectionsPerPortSsl =
         verifiableProperties.getIntInRange("router.scaling.unit.max.connections.per.port.ssl", 2, 1, 100);
-    routerConnectionsWarmUpPercentagePerPort =
-        verifiableProperties.getIntInRange("router.connections.warm.up.percentage.per.port", 25, 0, 100);
+    routerConnectionsLocalDcWarmUpPercentage =
+        verifiableProperties.getIntInRange("router.connections.local.dc.warm.up.percentage", 25, 0, 100);
+    routerConnectionsRemoteDcWarmUpPercentage =
+        verifiableProperties.getIntInRange("router.connections.remote.dc.warm.up.percentage", 0, 0, 100);
```

Comment:
Should the default value be 25 here? We are setting default in the declaration decorator to 25.

Reply:
ah thanks for catching that. The default should be 0 since I feel that warming up cross DC connections only applies to specific ambry use cases, but I will fix the decorator.